clientv3/concurrency/stm.go

```diff
@@ -73,12 +74,24 @@ func WithAbortContext(ctx context.Context) stmOption {
 	return func(so *stmOptions) { so.ctx = ctx }
 }
 
+// WithPrefetch is a hint to prefetch a list of keys before trying to apply.
```

Comment:
mention the advantage of prefetch a little bit here? when to use it for example?